### PR TITLE
Homework 29-04-24

### PR DESCRIPTION
**Pull request by alvardhovakimyan**
# Hospital Queries
### Description:

This pull request includes fixes for the SQL queries in the `hospital_queries.sql` file. The queries have been adjusted to ensure correct execution and results.

### Changes
Title: Update hospital_queries.sql

Description:
- Task 1: Changed column aliases for better readability and added quotation marks.
- Task 2: Fixed the unknown column error by adding appropriate column aliases for the Patient table.
- Task 3: Added meaningful aliases for the selected columns and included the department name.
- Task 4: Improved column aliases and added quotation marks for better readability.

### CODE REVIEW

File: Alvard/databases/29-04-24/hospital_queries.sql [hunk 1]
```diff
@@ -0,0 +1,28 @@
+-- Task 1: Choose the address and name of the hospital, the name of the department, the name of the doctor, and the name of the patient who underwent a specific operation
+SELECT h.address, h.name AS hospital_name, d.name AS department_name, doc.name AS doctor_name, p.name AS patient_name
```
Comment (elya-im) [inline Alvard/databases/29-04-24/hospital_queries.sql]: You can use the aliases with spaces like this, this version is more readable
``` 
select h.address as "Hospital Address"
           ,h.name as "Hospital name"
           ,d.name as "Department name"
           ,dc.name as "Doctor name"
           , p.name as "Patient name" 
```

File: Alvard/databases/29-04-24/hospital_queries.sql [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+FROM Hospital h
+JOIN Department d ON h.id = d.hospital_id
+JOIN Doctor doc ON d.id = doc.department_id
+JOIN Appointment a ON doc.id = a.doctor_id
+JOIN Patient p ON a.patient_id = p.id
+JOIN Invoice i ON p.id = i.patient_id
+WHERE i.description = 'heart surgery';
+
+-- Task 2: Select the name, surname, and phone number of the patients who have an appointment for the given month
+SELECT p.name, p.phone_number
```
Comment (elya-im) [inline Alvard/databases/29-04-24/hospital_queries.sql]: use meaningful aliases for the selected columns to make the result readable

File: Alvard/databases/29-04-24/hospital_queries.sql [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+FROM Patient p
+JOIN Appointment a ON p.id = a.patient_id
+WHERE MONTH(a.date) = 5; -- Assuming May is the given month
+
+-- Task 3: Choose the names and surnames of doctors appointed for each department
+SELECT d.name AS department_name, doc.name AS doctor_name
```
Comment (elya-im) [inline Alvard/databases/29-04-24/hospital_queries.sql]: - use meaningful aliases for the selected columns to make the result readable
- add department name

File: Alvard/databases/29-04-24/hospital_queries.sql [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+FROM Department d
+JOIN Doctor doc ON d.id = doc.department_id;
+
+-- Task 4: Select the personal information and name of the pharmacy for patients whose prescriptions cost $500 or more
+SELECT p.*, ph.name AS pharmacy_name
```
Comment (elya-im) [inline Alvard/databases/29-04-24/hospital_queries.sql]: use meaningful aliases for the selected columns to make the result readable

File: Alvard/databases/29-04-24/hospital_queries.sql (hunk 1 above)
Comment (alvardhovakimyan) [reply Alvard/databases/29-04-24/hospital_queries.sql]: ok,  i fixed it

**Review by elya-im — APPROVED** [review]

**Pull request merged**